### ISSUE TITLE
Docs for Url.Parser.Query.custom mention non-existing List.maybeMap

### ISSUE THREAD

**Issue by Janiczek** [elm/url#27]
https://package.elm-lang.org/packages/elm/url/latest/Url-Parser-Query#custom
This is the example in the documentation:
```elm
posts : Parser (Maybe (List Int))
posts =
  custom "post" (List.maybeMap String.toInt)

-- ?post=2        == [2]
-- ?post=2&post=7 == [2, 7]
-- ?post=2&post=x == [2]
-- ?hats=2        == []
```
`List.maybeMap` doesn't exist.

**Comment by Natim** [comment]
`maybeMap = List.filterMap identity` ?

**Comment by sethlivingston** [comment]
The correct function is [List.filterMap](https://package.elm-lang.org/packages/elm/core/latest/List#filterMap). The PR fixes the docs.

**Comment by evancz** [comment]
Fixed in #28

**Comment by PixelPartner** [comment]
The current doc still mentions `List.maybeMap`
